ml-agents-envs/mlagents_envs/rpc_utils.py

```diff
@@ -110,6 +110,26 @@ def _process_visual_observation(
     return np.array(batched_visual, dtype=np.float32)
 
 
+def _raise_on_nan_and_inf(data: np.array, source: str) -> np.array:
+    # Check for NaNs or infs in the observations
+    # If there's a NaN in the observations, the np.mean() result will be NaN
+    # If there's an Inf (either sign) then the result will be Inf
+    # See https://stackoverflow.com/questions/6736590/fast-check-for-nan-in-numpy for background
+    # Note that a very large values (larger than sqrt(float_max)) will result in an Inf value here
```

Comment:
This comment isn't correct anymore right? It will throw a RuntimeError

Reply:
yes, updated it.